src/components/app/AppHeader.js

```diff
@@ -0,0 +1,76 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+//
+import * as React from 'react';
+
+import { setDataSource } from 'firefox-profiler/actions/profile-view';
+import explicitConnect, {
+  type ConnectedProps,
+} from 'firefox-profiler/utils/connect';
+
+import './AppHeader.css';
+
+type DispatchProps = {|
+  +setDataSource: typeof setDataSource,
+|};
+type Props = ConnectedProps<{||}, {||}, DispatchProps>;
+
+class AppHeaderImpl extends React.PureComponent<Props> {
```

Comment:
Please add some documentation to this component, about why it exists and how it should be used.